README.md

```diff
@@ -1,2 +1,91 @@
-# armada
+# Armada
 A python orchestrator for a installing, upgrading, and managing a collection of helm charts, dependencies, and values overrides.
+
+Note that this project is pre-alpha and under active development. It may undergo drastic changes to support the long-term vision but contributions are welcome. 
+
+# Overview
+
+TThe armada python library and command line tool provides a way to synchronize an operators intended state, consisting many several charts, dependencies, and overrides using a single file, with an intended helm target.  This allows operators to define many charts, different namespaces for those releases, and overrides in a central place and with a single command, deploy and upgrade them all where applicable.
```

Comment:
Not that I'm qualified to comment on any of this... but did notice the `TThe` at the front of this sentence.